LocalWiki.php

```diff
@@ -200,6 +200,16 @@ function onClosedSiteNoticeAfter( &$siteNotice, $skin ) {
 	$wgGroupPermissions['*']['createtalk'] = false;
 }
 
+if ( $wgDBname === 'sau226wiki' ) {
+	$wgGroupPermissions['sysop']['abusefilter-modify-restricted'] = false;
+	$wgGroupPermissions['sysop']['editinterface'] = false;
+	$wgGroupPermissions['sysop']['globalblock-whitelist'] = false;
+	$wgGroupPermissions['sysop']['ipblock-exempt'] = false;
+	$wgGroupPermissions['sysop']['nuke'] = false;
+	$wgGroupPermissions['bureaucrat']['managewiki'] = false;
+	$wgGroupsRemoveFromSelf['consul'][] = 'consul';
```

Comment:
Please do this in LocalSettings.php as well.

Reply:
Sorry but I need a bit of assistance regarding what code to use for wgGroupsRemoveFromSelf . Please also see my comment above